converters/fromZigbee.js

```diff
@@ -525,6 +525,13 @@ const converters = {
             return {temperature: calibrateAndPrecisionRoundOptions(temperature, options, 'temperature')};
         },
     },
+    device_temperature: {
+        cluster: 'genDeviceTempCfg',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            return {temperature: parseInt(msg.data['currentTemperature'])};
```

Comment:
Change `temperature` -> `device_temperature`

Reply:
Done. 
Also I've added `sensor_device_temperature` to `homeassistant.js`